startGame.js

```diff
@@ -0,0 +1,40 @@
+document.getElementById("button").addEventListener('click', function(){
+    timer()
+    hideStartBtn()
+    displayGameScreen()
+})
+
+function hideStartBtn() {
+        document.getElementById("button").style.display = "none"
+    }
+function displayGameScreen() {
+    document.getElementById("gameScreen").style.display = "block"
+}
+
+
+function countDown(number) {
+    return number - 1
+}
+
+var startTimer = document.getElementById("timer")
+
+function createTimer() {
+    var timing = setInterval(function () {
+        if(time === 1){
+            clearInterval(timing)
+            startTimer.textContent = 'Go!!'
+        } else {
+            time = countDown(time)
+            startTimer.textContent = time
+        }
+    },1000)
+}
+
+function timer() {
+    time = 4
+    var startTimer
```

Comment:
unused variable. What is this for? You are calling a named function within anohter function. Not sure what this function realy does. Still n o documentation?